vue/src/TagmanagerTrackingCode/TagManagerTrackingCode.less

```diff
@@ -21,4 +21,30 @@
   .siteSelector {
     max-width: 100%;
   }
+
+  .trackingCodeAdvancedOptions {
+    margin-left: 20px;
+
+    > .advance-option {
+      margin: 1rem 0;
+    }
+
+    ul {
```

Comment:
Maybe this `ul` selector could be changed to `#mtm-advanced-options > ul` since the Website dropdown also contains a `ul` element and the margin-bottom is being applied to that as well:
![image](https://github.com/matomo-org/tag-manager/assets/1289711/903f2951-12e6-4196-bd64-3b47cece9ba5)

Reply:
Good catch. It was because I used the browser-default class from core, but it applies to any children. I removed the class and copied what it does into my scoped CSS.